### PR TITLE
Add issue platform infra

### PR DESCRIPTION
**Pull request by hubertdeng123**
Yes, I know...more containers. 😬

adds issue platform infra for https://github.com/getsentry/self-hosted/issues/2269

### CODE REVIEW

File: install/create-kafka-topics.sh
Comment (omjaz909) [inline install/create-kafka-topics.sh:6]: Fix

**Review by omjaz909 — COMMENTED** [review]
Running all appliances

**Comment by hubertdeng123** [comment]
I have this working now.
Screenshot 2023-08-02 at 2 33 32 PM

**Comment by hubertdeng123** [comment]
Needs https://github.com/getsentry/arroyo/pull/275

or else arroyo will complain

**Review by azaslavsky — APPROVED** [review]
So this LGTM, but tbh I don't know much about the setups for these services. I think it'd be best to get reviews from folks on the Snuba, Ingest, and Issues teams before merging.

**Comment by hubertdeng123** [comment]
> I think it'd be best to get reviews from folks on the Snuba, Ingest, and Issues teams before merging.

Dameli and Dan from issues have looked over this PR, so going to call it good

**Pull request merged**
